manifests/t/tanshuai/alphabiz/0.1.0.0/tanshuai.alphabiz.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: tanshuai.alphabiz
+PackageVersion: 0.1.0.0

```

```suggestion
PackageVersion: 0.1.0
```
This must match Control Panel.